docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>Probably a minor formatting suggestion but it looks nice to start a sentence with Uppercase.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>pushed a fix for uppercasing those bullets</p>